src/PrestaShopBundle/Form/Admin/Sell/Product/Combination/CombinationItemType.php

```diff
@@ -98,12 +98,16 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 'icon' => 'mode_edit',
                 'attr' => [
                     'class' => 'edit-combination-item tooltip-link',
+                    'data-toggle' => 'pstooltip',
+                    'data-original-title' => $this->trans('Edit item', 'Admin.Actions'),
                 ],
             ])
             ->add('delete', IconButtonType::class, [
                 'icon' => 'delete',
                 'attr' => [
                     'class' => 'edit-combination-item tooltip-link',
+                    'data-toggle' => 'pstooltip',
+                    'data-original-title' => $this->trans('Delete item', 'Admin.Notifications.Warning'),
```

Comment:
```suggestion
                    'data-original-title' => $this->trans('Delete', 'Admin.Global'),
```
Same here, thank you! :-)